extensions/wikia/EditPageLayout/EditPageLayoutController.class.php

```diff
@@ -206,6 +210,20 @@ public function executeEditPage() {
 			? wfMessage( 'editpagelayout-notificationsLink-none' )->escaped()
 			: wfMessage( 'editpagelayout-notificationsLink', count( $this->notices ) )->parse();
 
+		try {
+			$templateType = ( new TemplateClassificationService() )
+				->getType( $wgCityId, $this->title->getArticleID() );
+		} catch ( Exception $e ) {
+			$templateType = TemplateClassificationService::TEMPLATE_UNKNOWN;
```

Comment:
@katanka Why do you need to set $templateType to TEMPLATE_UNKNOWN when exception occurred?
Wouldn't something like this achieve same effect in more straightforward way?
```
try {
    $templateType = ( new TemplateClassificationService() )->getType( $wgCityId, $this->title->getArticleID() );
    $this->showInfoboxPreview = $templateType == INFOBOX || $templateType == CUSTOM_INFOBOX;
} else {
    $this->showInfoboxPreview = false;
    Log('omg, something went wrong');
}
```